### ISSUE TITLE
Tutorial for Klados

### ISSUE THREAD

**Issue by gaurav** [phyloref/klados#227]
Given that we came up with a tutorial for Phyx.js, maybe we should make one for Klados as well? We could use screenshots from the program to demonstrate how it should be used. We've made these types of screenshots for presentations before, so this shouldn't be too hard to put together.

**Comment by hlapp** [comment]
Yes indeed. Also see my [comment](https://github.com/phyloref/klados/issues/112#issuecomment-973492066) on #112.

**Comment by gaurav** [comment]
Here's one way we could organize the Klados tutorial:
1. How to go from a free-text clade definition from a publication to a phyloreference in OWL using Klados.
2. Resolving the phyloreference against the Open Tree of Life and exporting the phylogeny as Newick.
3. How to transcribe a phylogeny into Klados and to test the resolution of the phyloreference.
4. (optional) Entering a second phyloreference and comparing its resolution with the first phyloreference on the same phylogeny.

Is there anything else we need to include?

**Comment by gaurav** [comment]
Should strictly follow the curator use-cases in the manuscript.

**Comment by gaurav** [comment]
- [ ] To formalize this a bit more, we could turn this into a list of steps that need to be taken to fulfill a user story.
